packages/eslint-plugin/src/rules/no-type-imports-with-verbatim-side-effects.ts

```diff
@@ -0,0 +1,78 @@
+import type { TSESLint, TSESTree } from '@typescript-eslint/utils';
+import { AST_NODE_TYPES } from '@typescript-eslint/utils';
+
+import * as util from '../util';
+
+type Options = [];
+type MessageIds = 'useTopLevelQualifier';
+
+export default util.createRule<Options, MessageIds>({
+  name: 'no-type-imports-with-verbatim-side-effects',
```

Comment:
This rule name is a mouthful. What do you think about this?

```suggestion
  name: 'no-type-imports-with-side-effects',
```

I think this is still accurate to what the rule does, but with a little less length of name?

Reply:
The reason I added `verbatim` is because this rule really only applies with `verbatimModuleSyntax: true`.
I didn't want people to get confused and think that it might apply in other setups.

But yeah the rule name was long. it kind of was a placeholder until I thought of something better then it just "stuck"